### PR TITLE
add/update host_new entities & views

### PR DESCRIPTION
**Pull request by amolpati30**
i have made some small changes and added one airgun entities and views for host_new module.
below test_case is depending on given changes.

Dependent PR: https://github.com/SatelliteQE/robottelo/pull/14556

### CODE REVIEW

File: airgun/entities/host_new.py [hunk 1]
```diff
@@ -42,6 +43,7 @@ def get_details(self, entity_name, widget_names=None):
         view.wait_displayed()
         self.browser.plugin.ensure_page_safe()
         # Run this read twice to navigate to the page and load it before reading
+        time.sleep(2)
```
Comment (omkarkhatavkar) [inline airgun/entities/host_new.py:46]: please remove the hard sleep and use the wait_for everywhere it needed.

File: airgun/views/host_new.py [hunk 2]
```diff
@@ -792,8 +793,12 @@ def read(self):
 class EditAnsibleRolesView(View):
     """Edit Ansible Roles Modal"""
 
-    ROOT = ''
-    # No current representation for this Widget in Widgetastic
+    # ROOT = ''
```
Comment (omkarkhatavkar) [inline airgun/views/host_new.py:796]: # ROOT is not required here

**Review by omkarkhatavkar — CHANGES_REQUESTED** [review]
Please remove the hard sleep and move that to wait_for

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_add_and_remove_ansible_role_in_new_ui
robottelo: 14556

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 232
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py -k test_positive_add_and_remove_ansible_role_in_new_ui --external-logging
Test Result : =========== 1 passed, 8 deselected, 9 warnings in 750.95s (0:12:30) ============
```

File: airgun/entities/host_new.py (hunk 1 above)
Comment (amolpati30) [reply airgun/entities/host_new.py:46]: i am working on it

File: airgun/views/host_new.py (hunk 2 above)
Comment (amolpati30) [reply airgun/views/host_new.py:796]: ok, i will remove

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_assign_and_remove_ansible_role_in_new_ui
robottelo: 14556

**Comment by omkarkhatavkar** [comment]
@amolpati30 Nice Work !

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**